create_pnor_image.pl

```diff
@@ -131,6 +131,7 @@
 $build_pnor_command .= " --binFile_SECBOOT $scratch_dir/secboot.bin.ecc";
 $build_pnor_command .= " --binFile_VERSION $openpower_version_filename";
 $build_pnor_command .= " --binFile_IMA_CATALOG $scratch_dir/ima_catalog.bin.ecc";
+$build_pnor_command .= " --binFile_WOFDATA $scratch_dir/wofdata.bin.ecc";
```

Comment:
Where is this file going to come from?  We probably shouldn't merge this commit until we have a file in hostboot-binaries if that is where it comes from.  Otherwise, the daily "auto update" will be broken until you do that part.